src/Disks/IO/CachedOnDiskReadBufferFromFile.h

```diff
@@ -129,19 +129,7 @@ class CachedOnDiskReadBufferFromFile : public ReadBufferFromFileBase
 
     ReadType read_type = ReadType::REMOTE_FS_READ_BYPASS_CACHE;
 
-    static String toString(ReadType type)
-    {
-        switch (type)
-        {
-            case ReadType::CACHED:
-                return "CACHED";
-            case ReadType::REMOTE_FS_READ_BYPASS_CACHE:
-                return "REMOTE_FS_READ_BYPASS_CACHE";
-            case ReadType::REMOTE_FS_READ_AND_PUT_IN_CACHE:
-                return "REMOTE_FS_READ_AND_PUT_IN_CACHE";
-        }
-        UNREACHABLE();
-    }
+    static String toString(ReadType type) { return String(magic_enum::enum_name(type)); }
```

Comment:
Let's move the it somehow to cpp file (not sure you can do it for static methods though). magic_enum is super heavy to be present in a header.

Reply:
Will do that, tnx.